### ISSUE TITLE
install.vbs fails to copy react.* files into app/js on Windows

### ISSUE THREAD

**Issue by JimBTek** [forcedotcom/SmartSyncExplorerReactNative#56]
I did two attempts, one with cmd.exe running as admin. both tims install.vbs executed everything except copying the react.\* js files to app/js 

Manually copying the files resolved the issue

**Comment by helenren** [comment]
@JimBTek, I am investigating the issues that you report. Could you please supply following information that help me to dig into:
1) what's the Windows version that you use that you hit the issue?
2) What do you plan to do using the code from this repo? Just run this react native app on Windows?
3) Since we only supply the app for Android and iOS so far, the app you tried is created by yourself? 

Thanks.

**Comment by JimBTek** [comment]
1) Windows Version: 

Edition:  10
Version: 1511
OS Build: 10586.318  (I don't have the new 10.5 or whatever yet with all the new dev tools)  

2) We are building a React Native app for most likely iOS use, and possibly Android.  I wanted to build it to test on Android (which I do have successfully running now).  

3) All I did was clone the repo and follow the instructions in the readme.  It has to be built before it can be opened in Android Studio.  Install.vbs lines 47-55 attempts to copy:
 `("copy external\shared\libs\react.* app\js\", 1, True)`
But it fails to do so I believe because  `external\shared\libs\react.*`
Is not formatted in a way that works on Windows.

This is the same as the other Issue #55 I reported where your Package.json also fails on Windows without modification of the directory command.
